mkosi/distributions/fedora.py

```diff
@@ -145,6 +145,11 @@ def invoke_dnf(
         "--noplugins",
     ]
 
+    # Make sure we download filelists so all dependencies can be resolved.
+    # See https://bugzilla.redhat.com/show_bug.cgi?id=2180842
```

Comment:
That bug seems quite confused. Did you encounter some issue with mkosi or is this just a precaution?

Reply:
Yeah, I think this patch can be dropped. After the latest updates, it's just a bunch of fringe packages. `dnf5` is not even installed on older releases, so we don't have to worry so much about it.

Reply:
@keszybz Does that include all the centos packages? I still need to build centos 8 with mkosi.

Reply:
Ah, OK. I didn't think about the case where you use `dnf5` from the host to install an older distro. So let's merge this, and then maybe we can skip it again on Fedora >= 38. But that can be done later.